build.ps1

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env pwsh
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+[CmdletBinding()]
+param(
+    [switch]
+    $Clean,
+
+    [switch]
+    $Bootstrap,
+
+    [switch]
+    $Test,
+
+    [switch]
+    $NoBuild,
+
+    [switch]
+    $Deploy,
+
+    [string]
+    $CoreToolsDir,
+
+    [string]
+    $Configuration = "Debug",
+
+    [string]
+    $BuildNumber = '0'
+)
+
+#Requires -Version 6.2
```

Comment:
This should be 7.2 (we should use the latest version we hosted in Azure Functions).

Reply:
Done